components/raft_log_engine/Cargo.toml

```diff
@@ -28,5 +28,6 @@ serde = "1.0"
 serde_derive = "1.0"
 kvproto = { git = "https://github.com/pingcap/kvproto.git", default-features = false }
 raft = { version = "0.6.0-alpha", default-features = false }
-raft-engine = { git = "https://github.com/tikv/raft-engine"}
+# raft-engine = { git = "https://github.com/tikv/raft-engine"}
+raft-engine = { git = "https://github.com/hicqu/raft-engine", branch = "rewrite-batch-fix" }
```

Comment:
Can this branch get merged to the tikv org repository?

Reply:
Fixed.